Gruvo/Gruvo/Controllers/HomeController.cs

```diff
@@ -4,6 +4,10 @@
 using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
+using System.Net;
```

Comment:
why is this in commit? If changes don't affect current issue - they should not exist